src/qt/guiutil.cpp

```diff
@@ -710,7 +710,7 @@ QString formatServicesStr(quint64 mask)
     if (strList.size())
         return strList.join(" & ");
     else
-        return QObject::tr("None");
+        return QObject::tr("None", "absence of value");
```

Comment:
Our goal is to replace useless (in most cases) disambiguation strings (the second argument in `tr` call) with translator comments:
```suggestion
        //: Absence of value.
        return QObject::tr("None");
```